### DISCUSSION TITLE
Pimcore Academy: Stop/pause carousels

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#9528] [Q&A / Support]
Is there a way to pause or stop them? I can't finish reading them before they rotate, the height difference is also causing them to jump up and down on the page. Quite jarring! 😵

**Comment by dvesh3** [comment]
@NiklasBr which course are you looking at? the slides are still for me.

**Comment by NiklasBr** [comment]
Here for example on the What's New in Pimcore 6.7:

https://user-images.githubusercontent.com/279826/123063595-dbaacf80-d40d-11eb-94be-e242ff8959d3.mp4
